exodus/search/templates/search_form.html

```diff
@@ -30,7 +30,7 @@ <h4>{% trans "Search a report" %}</h4>
         <div class="col-6 col-md-4">
           <button id="go" class="btn btn-secondary btn-block">{% trans "Let's go!" %}</button>
         </div>
-      </div>
+      </div> 
```

Comment:
If you can remove the extra space here too :)

Reply:
Done!